### ISSUE TITLE
v0.4.4 fails to compile from source on MacOS

### ISSUE THREAD

**Issue by gobwas** [neovim/neovim#13157]
- Operating system/version: MacOS Catalina 10.15.6 (19G2021)
- CMake version: 3.17.1

### Steps to reproduce
```
git clone https://github.com/neovim/neovim.git
cd neovim
git checkout v0.4.4
make CMAKE_BUILD_TYPE=RelWithDebInfo
```

### Actual behaviour
```
[ 55%] Performing install step for 'libvterm'
CC bin/vterm-ctrl.c
bin/vterm-ctrl.c:117:10: error: implicit declaration of function 'strdup' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  return strdup(csi);
         ^
bin/vterm-ctrl.c:117:10: note: did you mean 'strcmp'?
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include/string.h:77:6: note: 'strcmp' declared here
int      strcmp(const char *__s1, const char *__s2);
         ^
bin/vterm-ctrl.c:117:10: warning: incompatible integer to pointer conversion returning 'int' from a function with result type 'char *' [-Wint-conversion]
  return strdup(csi);
         ^~~~~~~~~~~
bin/vterm-ctrl.c:142:10: error: implicit declaration of function 'strdup' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  return strdup(dcs);
         ^
bin/vterm-ctrl.c:142:10: warning: incompatible integer to pointer conversion returning 'int' from a function with result type 'char *' [-Wint-conversion]
  return strdup(dcs);
         ^~~~~~~~~~~
bin/vterm-ctrl.c:129:10: warning: result of comparison of constant 156 with expression of type 'char' is always false [-Wtautological-constant-out-of-range-compare]
    if(c == 0x9c) // ST
       ~ ^  ~~~~
3 warnings and 2 errors generated.
make[4]: *** [bin/vterm-ctrl] Error 1
make[3]: *** [build/src/libvterm-stamp/libvterm-install] Error 2
make[2]: *** [CMakeFiles/libvterm.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [deps] Error 2
```

### Expected behaviour
Successful compilation.

**Comment by gobwas** [comment]
However, if I checkout the _nightly_ tag, build succeeds.

**Comment by bfredl** [comment]
This appears to be an issue with libvterm @leonerd

**Comment by jamessan** [comment]
One which was already fixed in v0.1.4 and we've bumped the dep on nightly/release-0.4.  It will be fixed in 0.4.5.  In the meantime, you can use the pre-built version, install libvterm manually, or cherry-pick https://github.com/neovim/neovim/commit/20070310a43a8ddb76ea67bd4c50aa7c4dce54f9.
